### PR TITLE
[lldb] Fix few easily correctable warnings

### PR DESCRIPTION
**Pull request by kastiglione**

### CODE REVIEW

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -8342,14 +8342,6 @@ bool SwiftASTContext::CacheUserImports(SwiftASTContext &swift_ast_context,
                                        Status &error) {
   llvm::SmallString<1> m_description;
 
-  swift::ModuleDecl::ImportFilter import_filter {
```
Comment (kastiglione) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:8345]: @medismailben I believe this unused var is left over from your recent PR.

**Comment by kastiglione** [comment]
This PR is in prep for https://github.com/apple/llvm-project/pull/1991

**Comment by kastiglione** [comment]
@swift-ci test

**Comment by kastiglione** [comment]
@swift-ci test macOS Platform

**Review by adrian-prantl — APPROVED** [review]
Thanks!

**Comment by medismailben** [comment]
Thanks for catching that!

**Pull request merged**